ang/crmUi.js

```diff
@@ -256,6 +256,11 @@
           var init = function (retries, retryDelay) {
             var input = $('#' + id);
             if (input.length === 0) {
+              // CRM-21335 : there is no other way to fetch the 'ng-required' property of a subform (here recipients),
+              // so better to provide requiredness on basis of name which is a hackish fix for now.
+              if (attrs.crmUiFor === 'subform.recipients') {
+                scope.crmIsRequired = true;
+              }
```

Comment:
OK, I'm not quite there yet on accepting a hack for this. Let me try to find something a little bit cleaner. (And, if I can't then, we'll continue with this...)